lib/internal/bootstrap/node.js

```diff
@@ -761,30 +761,12 @@ function setupGlobalEncoding() {
 }
 
 function setupQueueMicrotask() {
+  const { queueMicrotask } =
+    NativeModule.require('internal/queue_microtask');
   Object.defineProperty(global, 'queueMicrotask', {
```

Comment:
We can do

```js
global. queueMicrotask = queueMicrotask;
```

now no?

Reply:
we can, but it's best to be explicit.

Reply:
We do it for `setTimeout`, `setImmediate`, etc. I'm fine with it as is though.

Reply:
Making it global means semver-major.

Reply:
it's already a global

Reply:
Ah! Right lol... Had forgotten that :)

Reply:
I personally prefer

```js
global.queueMicrotask = queueMicrotask;
```

It is significantly shorter and the implications should be clear.